oasis/models/iforest.py

```diff
@@ -155,6 +165,47 @@ def timeout_action(self):
             self.event.set()
 
 
+class IForestConfig(object):
+    def __init__(self, config=None):
+        self.global_ = dict()
+        self.metrics = dict()
+
+        # load config from config file
+        self._parse_config_file()
+
+        # load config from json data
+        if config is not None:
+            self._parse_api_config(config)
+
+        # set default value
+        self._set_default_config()
+
+    def _set_default_config(self):
+        if "train_count" not in self.global_:
+            self.global_["train_count"] = 120
```

Comment:
self.global_. setdefault("train_count", 120)

and you can use self._global instead of self.global_ if it is private var

Reply:
@zhouqiang-cl 
Use single quote (```) to wrap sensitive text.. Like `self.__global`. It is hard to know what you mean.

Reply:
you are right. I did not check it